### DISCUSSION TITLE
Error running the game:  “An RCT2 install directory must be specified!”

### DISCUSSION THREAD

**Discussion by zohar-fzh** [OpenRCT2/OpenRCT2#17579] [Q&A] [answered]
**When I entered "./openrct2", it says:  An RCT2 install directory must be specified! Please edit "game_path" in /home/zo/.config/OpenRCT2/config.ini.**
```
~/game/OpenRCT2/build$ ./openrct2
Importing legacy shortcuts...
An RCT2 install directory must be specified! Please edit "game_path" in /home/zo/.config/OpenRCT2/config.ini.
```

**And here`s my config.ini file.**
```
[general]
always_show_gridlines = false
autosave = 1
autosave_amount = 10
confirmation_prompt = false
currency_format = USD
custom_currency_rate = 10
custom_currency_affix = SUFFIX
custom_currency_symbol = "Ctm"
edge_scrolling = true
edge_scrolling_speed = 12
fullscreen_mode = 0
fullscreen_height = -1
fullscreen_width = -1
rct1_path = ""
game_path = "/home/zo/game/OpenRCT2/build"
```

**I`ve also tried using: _game_path = "/home/zo/game/OpenRCT2/build/usr"_ and _game_path = "/home/zo/game/OpenRCT2/build/data"_, but it does not seem to work.**

**Comment by janisozaur** [comment, marked as answer]
It looks like you tried setting it to _OpenRCT2_ directories, not _RCT2_. Please provide path to RCT2.

**Reply by zohar-fzh** [reply]
Thanks a lot, I got it, just like OpenDiablo2, that helps.
